modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -207,4 +208,32 @@ protected function getCartViews(array $carts) {
   public function getCacheMaxAge() {
     return 0;
```

Comment:
If we are adding cache contexts, we should not have a max age of `0`. We will also need tests on the cart's count to verify cache is busted.